opm/simulators/flow/FlowMainEbos.hpp

```diff
@@ -70,22 +66,21 @@ SET_INT_PROP(EclFlowProblem, OutputInterval, 1);
 
 END_PROPERTIES
 
+enum FileOutputMode {
```

Comment:
This is only used in flow.cpp now, so it can be moved there.

Reply:
OK